phpBB/adm/style/admin.css

```diff
@@ -1653,6 +1653,106 @@ fieldset.submit-buttons legend {
 	}
 }
 
+/* Mentions and mention dropdown
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+.atwho-view { /* mention-container */
+	text-align: left;
+	background-color: #ffffff;
+	border-radius: 2px;
+	box-shadow:
+		0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14),
+		0 1px 5px 0 rgba(0, 0, 0, 0.12);
+	position: absolute;
+	z-index: 999;
+	overflow: auto;
+	transition: all 0.2s ease;
+}
+
+.rtl .atwho-view { /* mention-container */
+	text-align: right;
+}
+
+.atwho-view-ul { /* mention-list */
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	color: #757575;
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 16px;
+	margin-left: 0;
+}
+
+.rtl .mention-media {
+	margin-right: 0;
+	margin-left: 16px;
+}
+
+.mention-media-avatar {
+	width: 40px;
+	height: 40px;
+}
+
+svg.mention-media-avatar { /* TODO: remove it after general normalization */
+	fill: currentColor;
+}
+
+.mention-item {
+	font-size: 16px;
+	font-weight: 400;
+	line-height: 1.5;
+	letter-spacing: 0.04em;
+	border-bottom: 1px solid #dddddd;
+	color: #212121;
+	position: relative;
+	display: flex;
+	overflow: hidden;
+	justify-content: flex-start;
+	align-items: center;
+	max-width: 300px;
+	padding: 16px;
+	cursor: pointer;
+}
+
+.mention-item:hover,
+.mention-item.cur {
+	text-decoration: none;
+	background-color: #eeeeee;
+	color: #2d80d2;
+}
+
+.mention-item:hover .mention-media-avatar,
+.mention-item.cur .mention-media-avatar {
```

Comment:
what is this for? why are you changing the avatar color on hover?

Reply:
That is for default avatars only.
You have this on your Codepen too.